tests/main/session-tool/task.yaml

```diff
@@ -44,3 +53,14 @@ restore: |
     # Remove files we've created
     rm -f /tmp/{inner,outer}.pid
     rm -f {before,after}.txt
+
+    # Restart background stuff we stopped.
+    sh defer.sh && rm -f defer.sh
```

Comment:
Could probably use `sh -x defer.sh` but we can tweak it anytime later.

Reply:
Yeah, good point.